adafruit_lsm9ds1.py

```diff
@@ -325,21 +333,23 @@ def temperature(self):
         temp = 27.5 + temp / 16
         return temp
 
-    def _read_u8(self, sensor_type, address):
+    def _read_u8(self, sensor_type: bool, address: int) -> None:
```

Comment:
Since this seems to be the template method, let's update the return to be `int` like the others.